lib/rules/no-irregular-whitespace/index.mjs

```diff
@@ -109,14 +111,14 @@ const rule = (primary) => {
 			validate(atRule, atRule.raws.before, zeroIndex);
 			validate(atRule, atRule.name, oneIndex);
 			validate(atRule, atRule.raws.afterName, atRuleAfterNameIndex);
-			validate(atRule, getAtRuleParams(atRule), atRuleParamIndex);
+			validate(atRule, cleanQuotedWhitespace(getAtRuleParams(atRule)), atRuleParamIndex);
 			validate(atRule, atRule.raws.between, atRuleBetweenIndex);
 			validate(atRule, atRule.raws.after, atRuleAfterIndex);
 		});
 
 		root.walkRules((ruleNode) => {
 			validate(ruleNode, ruleNode.raws.before, zeroIndex);
-			validate(ruleNode, getRuleSelector(ruleNode), zeroIndex);
+			validate(ruleNode, cleanQuotedWhitespace(getRuleSelector(ruleNode)), zeroIndex);
```

Comment:
I think it's required to use the parseSelector util for selectors. PostCSS Value Parser is for declaration values and at-rule params.